datasetinsights/io/config_handler.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import os
+import tempfile
+
+from yacs.config import CfgNode
+
+from datasetinsights.constants import GCS_BASE_STR
+from datasetinsights.io import create_downloader
+
+logger = logging.getLogger(__name__)
+
+
+class ConfigHandler:
```

Comment:
Why does this needs to be a python class? Can this method be a simple python function?